utils/helpers.py

```diff
@@ -18,3 +18,11 @@ def create_config_policy_resource_name(policy_prefix: str, resource_type:str) ->
 # Custodian prepends a prefix to resources.
 def create_deployed_config_policy_resource_name(policy_prefix: str, resource_type:str) -> str:
     return "custodian-" + create_config_policy_resource_name(policy_prefix, resource_type)
+
+
+def owner_tag_email_regex() -> str:
+    return "^((?!(@.*\\.)).)*$"
+
+
+def owner_tag_shared_regex() -> str:
+    return "(?i)^((?!shared).)*$"
```

Comment:
nit: consider defining these with constants instead of functions

Reply:
There's some overall refactoring that might be in order at this point. I'm going to move forward for now, but create a ticket to do some OOP changes down the line.